OracleWebLogic/dockerfiles/12.2.1.2/container-scripts/createAndStartEmptyDomain.sh

```diff
@@ -2,31 +2,6 @@
 #
 # Copyright (c) 2014-2015 Oracle and/or its affiliates. All rights reserved.
 #
-########### SIGTERM handler ############
```

Comment:
We still need a `SIGTERM` handler, because the `createAndStartEmptyDomain.sh` starts WebLogic and then tails the `AdminServer.log` file. Sending a CTRL+C inside the container (or telling it to stop) won't actually shutdown WebLogic.

You need to work out the correct way to stop WebLogic and then call that in `SIGTERM`.